ha/setup/create_pacemaker_resources.py

```diff
@@ -67,30 +154,35 @@ def change_pcs_default(cib_xml, push=False, **kwargs):
 
 def hax(cib_xml, push=False, **kwargs):
     """Create resources that belong to hax and clone the group."""
-    # hax
-    process.run_cmd(f"pcs -f {cib_xml} resource create hax systemd:hare-hax \
-        op start timeout=100s interval=0s \
+    hax_start = str(get_res_timeout(RESOURCE.HAX.value, TIMEOUT_ACTION.START.value, "hare-hax"))
+    hax_stop = str(get_res_timeout(RESOURCE.HAX.value, TIMEOUT_ACTION.STOP.value, "hare-hax"))
+    process.run_cmd(f"pcs -f {cib_xml} resource create {RESOURCE.HAX.value} systemd:hare-hax \
+        op start timeout={hax_start}s interval=0s \
         op monitor timeout=30s interval=30s \
-        op stop timeout=120s interval=0s")
-    process.run_cmd(f"pcs -f {cib_xml} resource clone hax")
+        op stop timeout={hax_stop}s interval=0s")
+    process.run_cmd(f"pcs -f {cib_xml} resource clone {RESOURCE.HAX.value}")
     if push:
         cib_push(cib_xml)
 
 def motr_conf(cib_xml, push=False, **kwargs):
-    process.run_cmd(f"pcs -f {cib_xml} resource create motr-confd-1 ocf:seagate:dynamic_fid_service_ra service=m0d fid_service_name=confd \
-        update_attrib=true \
-        op start timeout=100s interval=0s \
+    """Create resources that belong to motr confd and clone the group."""
+    confd_systemd = "m0d@" + get_fid("confd")
```

Comment:
Fid should not be required. This is unnecessary.
Try systemctl m0d status ...

Reply:
```
# systemctl status m0d
Unit m0d.service could not be found.
```
we need to use 
```systemctl status m0d@<fid>```
we can use random number like m0d@1 to know timeout
but get_fid is same providing any one fid for service
if to optimize 

we can hardcode fid to 1
```
def get_fid(service):
     return 1
```

Reply:
Updated code